services/auth/src/api.js

```diff
@@ -301,6 +336,34 @@ builder.declare({
   return res.reply(result);
 });
 
+builder.declare({
+  method: 'get',
+  route: '/audit/:entityType/:entityId',
+  query: {
+    ...paginateResults.query,
+  },
+  name: 'getEntityHistory',
+  category: 'Audit',
+  output: 'get-entity-history-response.yml',
+  scopes: 'auth:audit-history:<entityType>',
+  stability: 'stable',
+  title: 'Get Entity History',
+  description: [
+    'Get entity history based on entity type and entity name',
+  ].join('\n'),
+}, async function(req, res) {
```

Comment:
I think we need to be strict here about what types of entities this endpoint supports.
Someone might want to trigger this method for anything, like worker, queue, etc and if this doesn't return an error, it might look like there's just no entries. 

We can either add `params: { entityType: /(supported|list|entities)/ } ` 
or just check in the body first thing, and send back `404` no found or `400` invalid request argument. maybe 404 sounds better